org.eclipse.winery.yaml.common/src/test/java/org/eclipse/winery/yaml/common/reader/InvalidSyntaxTests.java

```diff
@@ -0,0 +1,20 @@
+package org.eclipse.winery.yaml.common.reader;
+
+import org.eclipse.winery.yaml.common.exception.MultiException;
+import org.eclipse.winery.yaml.common.reader.yaml.Reader;
+import org.junit.Test;
+
+/**
+ * Intention of this test class is to test the robustness of the parser
```

Comment:
For the record, this test case has nothing todo with the fix proposed in the PR. It was added because of a private conversation. Normally, this would go to a new, separate branch.

Reply:
For the record, this test case relates to changes done in ExceptionInterpreter.

Reply:
OK, now I see it. Maybe, the Content of the MultiExcpeption below should be checked somehow so that one sees the connection to the YAMLParserException. Can you do something like it was proposed at https://stackoverflow.com/a/13050567/873282?

Reply:
output of exception is:

```
org.eclipse.winery.yaml.common.exception.MultiException: mapping values are not allowed here
 in 'reader', line 1, column 61:
     ... osca_simple_yaml_1_0 description: test whether parser can handle ... 
                                         ^

null
```

There's no underlying `YAMLParserException` - and I don't think it's appropriate to test for the string "mapping values are not allowed", as this is a fragile construction

Reply:
Thank your for the explanation. I will add it as comment to the test so that others can work on it.